docs/docs/monitoring/analytics/example-queries.mdx

```diff
@@ -0,0 +1,194 @@
+---
+id: example-queries
+sidebar_label: Example queries
+title: Example queries
+description:
+---
+
```

Comment:
TODO: add MDX comment to explain how to get the graphs from https://rasa-technologies-gmbh.metabaseapp.com/dashboard/3-example-queries-for-the-docs